apps/frontend/src/pages/InProductsPage.jsx

```diff
@@ -1,22 +1,56 @@
 import Button from "../components/Button";
 import { IoMdBarcode } from "react-icons/io";
 import SearchBar from "../components/SearchBar";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import InOutProduct from "../components/InOutProduct";
 import { IoMdAdd } from "react-icons/io";
 import { MdOutlineCancel } from "react-icons/md";
 import { useNavigate } from "react-router-dom";
-import { getProductByCode, updateStocks } from "../services/ProductService";
+import { getProductByCode, getProducts, updateStocks } from "../services/ProductService";
 import { useUser } from "../hooks/UserUser";
 
 export default function InProductPage() {
   const [products, setProducts] = useState([]);
+  const [total, setTotal] = useState(0);
+  const [productsSearchList, setProductsSearchList] = useState([]);
   const navigate = useNavigate();
   const { user } = useUser();
 
-  const onSearch = async (code) => {
-    if (!code) return;
```

Comment:
Should keep this line, otherwise it will return a product even when you don't enter a code